OSVROpenGL/app/src/main/jni/main.cpp

```diff
@@ -652,6 +626,12 @@ void stop() {
         delete gClientContext;
         gClientContext = nullptr;
     }
+
+    if(gServer) {
+        LOGI("[OSVR] Shutting down server...");
+        gServer->stop();
```

Comment:
probably want to wait for stop, since `stop()` just sets the stop flag, doesn't mean the server thread received it or has actually stopped running yet.